ks-svarut/src/main/java/no/difi/meldingsutveksling/ks/SvarUtService.java

```diff
@@ -0,0 +1,23 @@
+package no.difi.meldingsutveksling.ks;
+
+import no.difi.meldingsutveksling.core.EDUCore;
+import org.springframework.stereotype.Service;
+
+@Service
+public class SvarUtService {
+    public EDUCoreConverter messageConverter;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make messageConverter a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)